src/CMake/SetUpThirdParty.cmake

```diff
@@ -573,14 +571,12 @@ endif()
 include(${VISIT_SOURCE_DIR}/CMake/FindVisItVTK.cmake)
 
 # Configure PySide Support
-if(VTK_VERSION VERSION_EQUAL "8.1.0")
-  if(VISIT_PYTHON_SCRIPTING AND PYTHONLIBS_FOUND
-                          AND NOT VISIT_DBIO_ONLY
-                          AND NOT VISIT_ENGINE_ONLY
-                          AND NOT VISIT_SERVER_COMPONENTS_ONLY)
-    include(${VISIT_SOURCE_DIR}/CMake/FindPySide.cmake)
-  endif()
-endif()
+#if(VISIT_PYTHON_SCRIPTING AND PYTHONLIBS_FOUND
+#                          AND NOT VISIT_DBIO_ONLY
+#                          AND NOT VISIT_ENGINE_ONLY
+#                          AND NOT VISIT_SERVER_COMPONENTS_ONLY)
+#    include(${VISIT_SOURCE_DIR}/CMake/FindPySide.cmake)
+#endif()
```

Comment:
Why commented out? Do we need this still?

Reply:
The if test for VTK 8 basically meant it was commented out for VTK 9. Probably should have an if-test based on Qt/Python versions instead. Can remove it and add a new ticket to see if we want to revive pyside.

Reply:
Created an issue, and added a new comment. Is this sufficient?